Gcp-ace-section4-complete-guide.html

```
@@ -0,0 +1,1401 @@
+    <div class="compare-grid">
+      <div class="compare-bad">
+        <div class="compare-label">✕ アンチパターン：JSON キーを Secret にマウント</div>
+        <div class="compare-content">SA の JSON キーを Kubernetes Secret に保存してPodにマウント。キー漏洩リスク・ローテーション管理の煩雑さが問題。<strong>絶対に使わないこと。</strong></div>
+      </div>
+      <div class="compare-good">
+        <div class="compare-label">✓ 推奨：Workload Identity Federation for GKE</div>
+        <div class="compare-content">KSA と GSA を紐付けるだけ。JSON キー不要。メタデータサーバーから自動的に短期トークンを取得。GKE Autopilot では自動有効化。</div>
+      </div>
+    </div>
+
+    <pre style="position:relative;"><button class="copy-btn" onclick="copyCode(this)">Copy</button"><span class="comment"># Step 1: GKE クラスタで Workload Identity を有効化</span>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)